Doc/using/configure.rst

```diff
@@ -185,6 +185,15 @@ General Options
 
    .. versionadded:: 3.11
 
+.. cmdoption:: --disable-gil
+
+   Enables experimental support for running Python without the
+   :term:`global interpreter lock` (GIL).
```

Comment:
Should we add a warning admonition emphasising that this is experimental, in development, etc -- mainly to preempt a random twitter recommendation that 'No GIL has now been implemented! Rebuild your Python today with `--disable-gil`..."

Perhaps a fear that won't materialise, though!

A

Reply:
I'm okay with adding whatever text you think is helpful. I expect to revisit and update this later in the development cycle based on what's actually integrated.